benchexec/model.py

```diff
@@ -93,16 +95,51 @@ def load_task_definition_file(task_def_file):
     if not task_def:
         raise BenchExecException("Invalid task definition: empty file " + task_def_file)
 
-    if str(task_def.get("format_version")) not in ["0.1", "1.0"]:
+    format_version = str(task_def.get("format_version"))
+    if format_version not in ["0.1", "1.0", "1.1"]:
```

Comment:
The supported format versions could be refactored into a constant at the file start, to make the supported versions easy to find